Resources/Prototypes/Research/industrial.yml

```diff
@@ -100,6 +100,7 @@
   recipeUnlocks:
   - ThermomachineFreezerMachineCircuitBoard
   - GasRecyclerMachineCircuitboard
+  - AlertsComputerCircuitboard
```

Comment:
deltav comment

Reply:
Fixed.

Reply:
its 1 thing so it only needs a single `# DeltaV` after it, not begin and end

Reply:
Alright, fixed that. Hope the comment is alright in there as well, seems like the contributing.md suggests to have something at least.

Reply:
The hell can't I resolve this conversation for :S